piccolo_api/crud/endpoints.py

```diff
@@ -218,6 +219,7 @@ def __init__(
             }
         else:
             self._hook_map = None  # type: ignore
+        self.audit_log_table = audit_log_table or AuditLog
```

Comment:
There's no need to the `or` here - the user should be allowed to set it to `None`.

Reply:
But without that we cannot add records to the `AuditLog` table (or I doing something wrong). If the user does not want to use the `audit_logs` app, just does not register it in `APP_REGISTRY` and can use `PiccoloCRUD` without `audit_logs`.

Reply:
Even if people don't register the `audit_logs` app, `PiccoloCRUD` will still be trying to make database queries to the `AuditLog` table, so we need to be able to set it to `None` to prevent this. I think when people use `PiccoloCRUD` standalone within their apps they probably won't need the audit logs - it's mostly for Piccolo Admin.

In `create_admin` we could add an `enable_auditing` argument - any tables listed will have auditing switched on in `PiccoloCRUD`.

```python
def create_admin(..., audit_log_table: AuditLog = AuditLog, enable_auditing: t.List[t.Type[Table]] = [])
```

Reply:
Good point. Later today I will try to do that in Piccolo Admin.

Reply:
You are right about this. We need to add a few lines of code to Piccolo Admin and everything works great and we can choose which table we want audit logs for in future `enable_auditing` argument in `create_admin`.